doc/ug/getting_started_build_sw.md

```diff
@@ -95,6 +95,159 @@ $ ./meson_init -t /path/to/toolchain/file
 If you do not specify your own toolchain configuration file (using `./meson_init.sh -t`), and `meson_init.sh` cannot find the default configuration in your toolchain, the legacy `toolchain.txt` from the main OpenTitan repository will be used.
 If `TOOLCHAIN_PATH` is set, this will be used to update any paths within the legacy configuration.
 
+## Building Software with Bazel
+
+The Mask ROM and many other software targets are built with
+(Bazel)[https://bazel.build/]
+
+TLDR:
+after installing Bazel you can run
+
+```console
+bazel test //... --test_tag_filters=-cw310 --disk_cache=~/bazel_cache
+```
+
+### Installing Bazel
+
+If you haven't yet installed Bazel, add it to apt and instal it on ubuntu systems  with the following
+commands:
+
+```console
+sudo apt install apt-transport-https curl gnupg
+curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
+sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
+echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" |
```

Comment:
Might be worth mentioning somewhere that the jdk1.8 thing is a legacy thing and doesn't actually mean jdk1.8 is being used, just to stop people going down that rabbit hole.

Reply:
I've linked to the discussion upstream in the Bazel documentation. I hope that's sufficient.